### DISCUSSION TITLE
Server Side rendering not working for me

### DISCUSSION THREAD

**Discussion by mayank2108** [vercel/next.js#14189] [Help] [answered]
View source of https://cherishx.com. Only scripts tag are shown which includes chunks, pages and framework js. My divs, meta tags  and other render html is not visible. 

Due to this fb is also not able to detect open graph tags. Pls help how to get a full rendered html code in view source for my page.

**Comment by mayank2108** [comment]
I am using getServerSideProps to fetch data from external APIs.

**Comment by rafaelalmeidatk** [comment]
It is hard to point the exact issue without the source code. My guess is that the code is doing a conditional render, and this conditional render only renders anything after an `useEffect` is executed, which is something that doesn't happen in the server

**Comment by mayank2108** [comment, marked as answer]
Issue is resolved. I was using redux persist gate gate in my app.js which was preventing server side rendering to happen.
